README.md

```diff
@@ -36,14 +36,15 @@ npm install -g @asyncapi/generator
 
 ## Supported parameters
 
-| Name | Description | Required | Default | Allowed values |  Example |
+| Name | Description | Required | Default | Allowed values | Example |
 |---|---|---|---|---|---|
 | sidebarOrganization | Defines how the sidebar should be organized. Set its value to `byTagsNoRoot` to categorize operations by operations tags. Set its value to `byTags` when you have tags on a root level. These tags are used to model tags navigation and need to have the same tags in operations. | No | undefined | `byTags`, `byTagsNoRoot` | `byTagsNoRoot` |
 | baseHref | Sets the base URL for links and forms. | No | `/` | *Any* | `/docs` |
-| version | Override the version of your application provided under `info.version` location in the specification file. | No | Version is taken from the spec file. | *Any* ([See Semver versionning](https://semver.org/)) | `1.0.0` |
+| version | Override the version of your application provided under `info.version` location in the specification file. | No | Version is taken from the spec file. | *Any* ([See Semver versioning](https://semver.org/)) | `1.0.0` |
 | singleFile | Set output into one html-file with styles and scripts inside | No | `false` | `true`,`false` | `true` |
 | outFilename | The filename of the output file. | No | `index.html` | *Any* | `asyncapi.html` |
 | pdf | Generates output HTML as PDF | No | `false` | `true,false` | `false` |
+| config | Inline stringified JSON or a path to a JSON file to override default React component config | No | `null` | [JSON config for the React component](https://github.com/asyncapi/asyncapi-react/blob/next/docs/configuration/config-modification.md#definition) | `{"show":{"sidebar":false}}` |
```

Comment:
Default value for that parameter should be `{ show: { sidebar: true }, sidebar: { showOperations: 'byDefault' } }`. Also we should mention merging of config works:

```suggestion
| config | Inline stringified JSON or a path to a JSON file to override default React component config. The default config and passed via patameter is merged using the [JSON Merge Patch](https://tools.ietf.org/html/rfc7386) algorithm. | No | `{ "show": { "sidebar": true }, "sidebar": { "showOperations": 'byDefault' } }` | [JSON config for the React component](https://github.com/asyncapi/asyncapi-react/blob/next/docs/configuration/config-modification.md#definition) | `{"show":{"sidebar":false}}` |
```

Reply:
done